README.md

```diff
@@ -35,30 +35,30 @@ ng new my-app
 ng add @o3r/core
 ```
 
-> **Note**: Please refer to [Otter Get Started](./docs/core/START_NEW_APPLICATION.md) and [Angular Get Started](https://angular.io/guide/setup-local#install-the-angular-cli) documentations for complete documentation
+> **Note**: Please refer to [Otter Get Started](./docs/core/START_NEW_APPLICATION.md) and [Angular Get Started](https://angular.io/guide/setup-local#install-the-angular-cli) for complete documentation.
 
 ## Contributing
 
-Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
+Please read the [Contributing](./CONTRIBUTING.md) file for details on our code of conduct and the process to submit pull requests.
 
 ## Versioning
 
-Please refer to [Security file](./SECURITY.md).
+Our version management process is still in progress.
```

Comment:
the previous sentence was correct

Reply:
the link was correct, but the content of the `SECURITY` file is not.
We still support otter `6.x` until Angular drops the support of the major `13` in May.

Ideally this file should mirror what we have today on the internal wiki here: https://dev.azure.com/AmadeusDigitalAirline/DES%20Platform/_wiki/wikis/DES%20Documentation/3218/Actively-supported-versions